test/support/message_case.ex

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.MessageCase do
```

Comment:
Is this used in multiple places? Otherwise moving generation of Plain Old Data in a separate case adds an unnecessary level of indirection

Reply:
It is used here in `HTTPRequestsTest` and `WebSocketsTest`, and in later PRs it is used inside `ForwarderCase`